components/arm/fake/fake.go

```diff
@@ -81,8 +85,8 @@ func buildModel(cfg resource.Config, newConf *Config) (referenceframe.Model, err
 	case modelPath != "":
 		model, err = modelFromPath(modelPath, cfg.Name)
 	default:
-		// if no arm model is specified, we return an empty arm with 0 dof and 0 spatial transformation
-		model = referenceframe.NewSimpleModel(cfg.Name)
+		// if no arm model is specified, we return a fake arm with 1 dof and 0 spatial transformation
+		model, err = modelFromName(Model.Name, cfg.Name)
```

Comment:
@raybjork I'm okay with just showing a 1dof arms and adding a warning if the tests still all pass.

@martha-johnston, can you also add the dofbot, might as well while we're here/.

Reply:
what do you mean also add the dofbot? like if a user specifically adds dofbot as the arm-model in the fake config?

Reply:
added, let me know if that's what you meant

Reply:
I think whatever we want to make the default is probably fine.

Adding the Dofbot seems ok for now but we're going to have to remove all this logic when we take these drivers out of RDK

Reply:
the dofbot is already out of the rdk, it only exists as a kinematics file in fake arm, it has no driver or registered resource.